### PR TITLE
More realistic header

### LINKED ISSUE #1095: GMF apps - add realistic content in header
We could add the geomapfish logo and links to github in the desktop app.

### PR DESCRIPTION
**Pull request by sbrunner**
fix  #1095
demo: https://camptocamp.github.io/ngeo/improve-header/examples/contribs/gmf/apps/desktop_alt/?lang=fr

### CODE REVIEW

**Comment by pgiraud** [comment]
The desktop app logo doesn't appear to be a valid image file.

File: contribs/gmf/less/desktop.less [hunk 1]
```diff
@@ -38,7 +38,6 @@ header {
   left: 0;
   height: @topbar-height;
   z-index: @zindex-navbar-fixed;
-  .box-shadow(0px 2px 6px -1px rgba(0, 0, 0, 0.5));
```
Comment (pgiraud) [inline contribs/gmf/less/desktop.less]: Any reason why you removed this shadow?

File: contribs/gmf/apps/desktop_alt/index.html [hunk 2]
```diff
@@ -8,7 +8,8 @@
   <body>
     <header>
       <div>
-        Put your logos or menus here
+        <img src="image/logo.png" />
```
Comment (pgiraud) [inline contribs/gmf/apps/desktop_alt/index.html]: This path is not valid when hosted.

File: contribs/gmf/less/desktop.less (hunk 1 above)
Comment (sbrunner) [inline contribs/gmf/less/desktop.less]: Because it's not beautiful especially on the right ...

File: contribs/gmf/apps/desktop_alt/index.html (hunk 2 above)
Comment (sbrunner) [inline contribs/gmf/apps/desktop_alt/index.html]: Is it true or the image is just not copied?

File: contribs/gmf/less/desktop.less (hunk 1 above)
Comment (pgiraud) [inline contribs/gmf/less/desktop.less]: Can this be discussed and addressed in a different pull request?

File: contribs/gmf/apps/desktop_alt/index.html (hunk 2 above)
Comment (pgiraud) [inline contribs/gmf/apps/desktop_alt/index.html]: Indeed, probably not copied.

**Comment by fredj** [comment]
`contribs/gmf/apps/desktop/image/logo.png` file is corrupted

**Comment by sbrunner** [comment]
Strange it's a copy ..., I will copy it again ...

**Comment by sbrunner** [comment]
Fixed :-)
